packages/fxa-settings/src/styles/ctas.scss

```diff
@@ -60,6 +68,10 @@
   @apply text-base mt-6;
 }
 
+.cta-base-block {
+  @apply text-base mt-12;
```

Comment:
That works for me! My impression is that this might be a good trend to follow in general with our Tailwind styles -- does that match your understanding?

Reply:
Yeah, I think so. Once a component class has styles that we need to override, I think we should just remove that class from the component class list and pull out the utility classes we need and stick 'em on the component. I think some cases would warrant creating a new component class entirely depending on the use case but I think this one's a good example of the former.